assets/app/styles/_core.less

```diff
@@ -777,6 +795,46 @@ a.disabled-link {
   }
 }
 
+.modal-resource-edit {
+  .editor {
+    font-family: @font-family-monospace;
+    line-height: 1.5;
+    width: 100%;
+    min-height: 100px;
+    // Disable numeric highlighting in YAML because of bugs
+    .ace_constant.ace_numeric {
```

Comment:
guessing there is no way to make it specific to the source type? does it add any class for what source type is currently in the editor?